.claude/skills/query-performance-analysis/reference/performance-thresholds.md

```
@@ -0,0 +1,283 @@
+### Ideal Index Characteristics
+
+| Metric            | Target             | Notes                                 |
+| ----------------- | ------------------ | ------------------------------------- |
+| Index selectivity | >10% unique values | High cardinality = better selectivity |
+| Index size        | <30% of table size | Compact indexes are efficient         |
+| Index usage       | Used by queries    | Unused indexes should be dropped      |
+
+## Table Size Guidelines
+
+| Table Size        | Max Query Time | Strategy                                |
+| ----------------- | -------------- | --------------------------------------- |
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)